### DISCUSSION TITLE
Attempted to load a Vector Map, but failed. Falling back to Raster

### DISCUSSION THREAD

**Discussion by KirillAleshkin** [visgl/react-google-maps#456] [Q&A] [answered]
after implementing the map I get the following message

"Attempted to load a Vector Map, but failed. Falling back to Raster"

Has anyone encountered this problem?

     <APIProvider apiKey={'XXX'} libraries={['marker']}>
			
        <Map
				mapId={'bf51a910020fa25a'}
				defaultZoom={3}
				defaultCenter={{ lat: 0, lng: 0 }}
				gestureHandling={'greedy'}
				disableDefaultUI
				onCameraChanged={handleCameraChange}
				{...cameraProps}
			>		
			{markers.map((marker) => {
				return <MarkerComponent key={marker.id} {...marker}  />
			})}
                </Map>
			
        </APIProvider>
		</>

**Comment by usefulthink** [comment]
I remember seeing this error a lot, do you happen to own a MacBook with Intel CPU and AMD GPU?

**Reply by KirillAleshkin** [reply]
Processor:2.6 GHz 6-Core intel Core i7
Graphics: AMD Radeon Pro 5300M

Thank you for the answer!

**Reply by usefulthink** [reply, marked as answer]
Yep, I think that's the one I had as well. 
There seems to be a very rare GPU bug (somewhere between macOS, GPU drivers, OpenGL and Chrome) that only affects this specific combination and for some reason the Maps API decided to blacklist WebGL based rendering on that GPU (you will notice that it works when using the Intel iGPU (for example, when no external screen is connected and the device runs on Battery).

The only thing that worked for me was to switch the setting "Use ANGLE graphics backend" (chrome://flags/#use-angle) to "Metal", but since this isn't without problems, I used a separate [Chrome Canary](https://www.google.com/intl/en/chrome/canary/) installation for that.

**Reply by KirillAleshkin** [reply]
Thank you Martin for the answer and for your advices!

At the same time, I see that the map is working and rendering, I just see this error in the console. Could this somehow affect the production version in the future and not show the map with this CPU for some clients and should we be afraid of this? what do you think about it?

**Reply by usefulthink** [reply]
The map will still render, but you can't use the vector-based maps (and related features like heading/tilt, WebGlOverlayView, ...). It will fall back to raster maps. If you need the vector features, you'll have to test in another browser (or use a chrome version that uses the Metal backend).

**Reply by KirillAleshkin** [reply]
Thank you!
